src/test/resources/features/complaints/ComplaintsSearch.feature

```diff
@@ -1,33 +1,30 @@
-@ComplaintsSearch @Complaints
+@ComplaintsSearch @Complaints @Search
 Feature: Complaints Search
 
 #     UKVI COMPLAINTS
 
-  # HOCS-2838, HOCS-3036
   @COMPRegression @UKVIComplaints
-  Scenario Outline: User tests UKVI complaint case search criteria
+  Scenario: User tests UKVI complaints case search criteria
     Given I am logged into "CS" as user "COMP_USER"
-    When I navigate to the "Search" page
-    And I enter "<infoValue>" into the "<infoType>" search field in the "COMP" search configuration
+    When I generate a "COMP" case to validate search functionality
+    And I navigate to the "Search" page
+    And I enter "COMP" into the "Case Type" search field
+    And I enter "Sam McTester" into the "Correspondent Full Name" search field
+    And I enter "AB1 2CD" into the "Correspondent Postcode" search field
+    And I enter "SamMcTester@Test.com" into the "Correspondent Email Address" search field
+    And I enter "01/01/2001" into the "Complainant Date of Birth" search field
+    And I enter "Test entry for HO Reference" into the "Complainant Home Office Reference" search field
+    And I enter the current case reference into the Case Reference field on the search screen
     And I click the search button on the search page
-    Then I check that the search results have the correct "<infoType>"
-    Examples:
-      | infoType                          | infoValue                            |
-      | Case Type                         | COMP                                 |
-      | Case Type                         | COMP2                                |
-      | Correspondent Full Name           | Sam McTester                         |
-      | Correspondent Postcode            | AB1 2CD                              |
-      | Correspondent Email Address       | SamMcTester@Test.com                 |
-      | Complainant Date Of Birth         | 01/01/2001                           |
-      | Complainant Home Office Reference | Test entry for Home Office Reference |
-
-  # HOCS-2838
+    Then the created case should be the only case visible in the search results
+
   @COMPRegression @UKVIComplaints
-  Scenario: User can search for a UKVI complaint case by its case reference
+  Scenario: User can search for a stage 2 UKVI complaint case
     Given I am logged into "CS" as user "COMP_USER"
-    When I create a single "COMP" case
-    And I search for the case by its case reference
-    Then the created case should be the only case visible in the search results
+    When I navigate to the "Search" page
+    And I enter "COMP2" into the "Case Type" search field
+    And I click the search button on the search page
+    Then I check that the search results have the correct "Case Type"
```

Comment:
we have this for COMP2 but not BF2, is it required do you think? If so should it be added for other stage 2 case types?